functions.php

```diff
@@ -207,7 +207,8 @@ function twentytwenty_register_scripts() {
 
 		$js_dependencies = array( 'wp-dom-ready', 'wp-polyfill' );
 
-		wp_enqueue_script( 'twentytwenty-index', get_template_directory_uri() . '/assets/js/index.js', $js_dependencies, $theme_version, false );
+		wp_enqueue_script( 'twentytwenty-construct', get_template_directory_uri() . '/assets/js/construct.js', $js_dependencies, $theme_version, false );
```

Comment:
This is now pointing to the wrong file. Should I rename the index.js file to construct.js again?

Reply:
No I will fix it, I prefer index.js